packages/components/src/visually-hidden/index.js

```diff
@@ -1,46 +1,3 @@
-/**
- * External dependencies
- */
-import classnames from 'classnames';
-
-/**
- * WordPress dependencies
- */
-import { forwardRef } from '@wordpress/element';
-
-/**
- * Internal dependencies
- */
-import { renderAsRenderProps } from './utils';
-import { withNextComponent } from './next';
-
-/**
- * @template {keyof JSX.IntrinsicElements | import('react').JSXElementConstructor<any>} T
- * @typedef OwnProps
- * @property {T} [as='div'] Component to render, e.g. `"div"` or `MyComponent`.
- */
-
-/**
- * @template {keyof JSX.IntrinsicElements | import('react').JSXElementConstructor<any>} T
- * @typedef {OwnProps<T> & import('react').ComponentPropsWithRef<T>} Props
- */
-
-/**
- * VisuallyHidden component to render text out non-visually
- * for use in devices such as a screen reader.
- *
- * @template {keyof JSX.IntrinsicElements | import('react').JSXElementConstructor<any>} T T
- * @param {Props<T>} props
- * @param {import('react').Ref<any>} forwardedRef
- * @return {JSX.Element} Element
- */
-function VisuallyHidden( { as = 'div', className, ...props }, forwardedRef ) {
-	return renderAsRenderProps( {
-		as,
-		className: classnames( 'components-visually-hidden', className ),
-		...props,
-		ref: forwardedRef,
-	} );
-}
-
-export default withNextComponent( forwardRef( VisuallyHidden ) );
+export { default as VisuallyHidden } from './component';
+export { default } from './component';
```

Comment:
Why do we export the component twice?

Reply:
Some places were importing it as the default and others were importing it by name, so I figured this would minimize the number of changes elsewhere in the components package. I can switch to just one or the other if you prefer though (probably just the named export is best?)

Reply:
I think we historically started with default exports for components. Even if I personally would prefer named components now, I think it's probably better to stay consistent and use default export here.